### DISCUSSION TITLE
Profile icon menu is vertically aligned, and menu items are cut off

### DISCUSSION THREAD

**Discussion by SonoSooS** [community/community#129439] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Product Feedback

### Body

(there is no UI feedback category, so I'm forced to report it here)

Before this change, the profile icon menu was aligned to the icon. Now, for some reason it's vertically aligned to the center.

Why?

Please give option to put it back where it belongs, stickied to the top-right corner.  
I have to move my mouse too much to access my profile (or anything else really) in the menu, and that's not great when more mouse movement is physically painful.

Is there a way to opt out of this A/B testing? Is any accessibility testing getting done at all before these unwanted features are pushed onto us? Why does the UI always have to be changed for the worse?

**Comment by TayabGhafor** [comment]
Accessibility Settings:

Check if GitHub offers accessibility settings that could potentially adjust the UI layout or increase usability.
Consider using browser extensions or settings that allow customization of UI elements for improved accessibility.
Providing Feedback:

Utilize GitHub's official channels (like their support email or forums) to voice your feedback.
Constructive feedback detailing how the changes impact your usage, especially in terms of accessibility and usability, can influence future updates.
Or you can directly give feedback to the Githhub.

**Reply by SonoSooS** [reply]
I checked every setting, and there is no option to revert the profile icon menu to the top-right corner.

I get sent around in circles when trying to write a feedback, and eventually landed here. If you could link a *real* way to give feedback that doesn't just send me around in circles, I would appreciate that.

I'm already using browser extensions to work around some hitches, but it doesn't help that the randomized classnames keep changing every time, making my changes be a cat-and-mouse game, and me eventually giving up just by the sheer frequency I have to maintain my custom stylesheets.


I tried my best to stay constructive, but it's difficult to keep my composure if they make unwarranted *bad* changes out of the blue, *repeatedly*, without having an option to revert+feedback, or at least any feedback at all.  
I wish they did like they did with colorblind colorschemes (which are *still* in feature preview for some reason, even though I have been using it ever since it was available, and encountered zero issues), where you would get a notification dot about a new feature, *opt in*, try it out, and disable+feedback if it's bad, or make use of it if it's good.

**Reply by TayabGhafor** [reply]
have u checked the github support forum.
**Visit GitHub's Support Page**
Go to GitHub's official support page where you can find help articles and contact options.
**GitHub Support Link:** https://support.github.com/

**Reply by TayabGhafor** [reply]
I completely understand your frustration with the recent changes on GitHub. It can be really challenging when updates impact our workflows unexpectedly. Your suggestion about having opt-in notifications for new features, similar to the colorblind colorschemes, is spot on. It would definitely help users like us to try out new features at our own pace and provide valuable feedback directly if something doesn't quite work for us.

I'm glad to hear that the colorblind colorschemes have been working well for you. It’s a great example of how user-centric features can enhance our experience on GitHub.

I encourage you to share your thoughts directly with GitHub through their official support channels or community forums. They really do listen to user feedback, and your insights could help shape future improvements.

**Reply by SonoSooS** [reply]
I still don't have a support forum option, I've looked everywhere :(  
Clicking "feedback" or any other similar-looking option directly or indirectly redirects me right back here to these discussions.

I would actually join [the accessibility panel](https://github.com/orgs/community/discussions/128172), but it seems like it's for workers at big corp only, and I lost my job years ago, so it seems like I'm not eligible to join (the questionnaire asks for mandatory questions which I can not answer, as I'm not employed anymore).

So for all intents and purposes, seems like I'm left at the mercy of some employee randomly stumbling upon these posts of mine months later within the massive sea of spam and scam posts, and hopefully get *any* response at this point.

**Comment by ebndev** [comment]
Hi @SonoSooS, thank you for taking the time to share your thoughts1

Would you mind sharing a screenshot of what you're talking about? I want to make sure I'm correctly identifying your issue.

While we can’t guarantee immediate action on every piece of feedback, please know that we are attentive to your suggestions in the GitHub Discussions space.

**Reply by SonoSooS** [reply]
It got changed to this:
![image](https://github.com/community/community/assets/7823099/1dd557d6-beba-42d2-9373-1703110f83ee)

It also gets cut off at different scroll levles (while reading code in multiple windows):
![image](https://github.com/community/community/assets/7823099/35ef095e-1371-42d0-a1d3-8d5519f6c10b)

It's not scrollable like the previous one was.
